docs/pages/docs/guides/document-fields.mdx

```diff
@@ -568,42 +617,19 @@ fields.object({
 
 #### Relationship Fields
 
-To use relationship fields on component blocks, you need to add a relationship to the document field config.
-
-```tsx
-import { config, list } from '@keystone-6/core';
-import { document } from '@keystone-6/fields-document';
-
-export default config({
-  lists: {
-    ListName: list({
-      fields: {
-        fieldName: document({
-          relationships: {
-            featuredAuthors: {
-              kind: 'prop',
-              listKey: 'Author',
-              selection: 'id title',
-              many: true,
-            },
-          },
-          /* ... */
-        }),
-        /* ... */
-      },
-    }),
-    /* ... */
-  },
-  /* ... */
-});
-```
-
-You can reference the key of the relationship in the relationship and in the form, it will render a relationship select like the relationship field on lists.
+To use relationship fields on component blocks, you need to add a relationship field and provide a list key, label and options selection.
+In the form, it will render a relationship select like the relationship field on lists.
+Similarly to inline relationships, they will be hydrated with this selection if `hydrateRelationships: true` is provided when fetching.
 
 ```tsx
 import { fields } from '@keystone-6/fields-document/component-blocks';
 
-fields.relationship({ label: 'Authors', relationship: 'featuredAuthors' });
+fields.relationship({
```

Comment:
Could you put this in context of what the surrounding code might look like,  `...` is fine